apps/client/src/intros/shared/DynamicIMTRIntro.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { autofillMap } from "../../config/autofill";
+import { Intro } from ".";
+
+type Props = {
+  checker: any;
```

Comment:
Does this make sense? 👀

Reply:
Added comment to make the intention more clear.